azure-pipelines.yml

```diff
@@ -3,7 +3,7 @@ trigger:
 - master
 - master-vs-deps
 - release/*
-- features/*
+- feature/convertNameOf
```

Comment:
why remove `features/*` ?

Reply:
whoops!

Reply:
@jmarolf: can this branch be called "features" instead of "feature" -- we have the naming convention to avoid having to update tings like this.